src/views/preview/presentation.jsx

```diff
@@ -277,23 +281,51 @@ const PreviewPresentation = ({
                                 ))}
                             </FlexRow>
                         </FlexRow>
-                        <FlexRow className="preview-row">
-                            <div className="comments-container">
-                                <div className="project-title" />
-                            </div>
-                            <FlexRow className="column">
-                                <RemixList remixes={remixes} />
-                                <StudioList studios={studios} />
+                    </div>
+                    <div className="project-lower-container">
+                        <div className="inner">
+                            <FlexRow className="preview-row">
+                                <div className="comments-container">
+                                    <FlexRow className="comments-header">
+                                        <h4>Comments</h4>
+                                        {/* TODO: Add toggle comments component and logic*/}
+                                        <div>comments on/off</div>
+                                    </FlexRow>
+                                    <FlexRow className="create-comment comment">
+                                        <Avatar src="https://cdn2.scratch.mit.edu/get_image/user/3581881_60x60.png" />
+                                        <ComposeComment />
+                                    </FlexRow>
+                                    <FlexRow className="comments-list">
+                                        {comments.map(comment => (
+                                            <CommentContainer
+                                                {...comment}
```

Comment:
We should list these props explicitly because otherwise API changes are implicitly passed as props — it will be easier to handle changes like those if we list what we expect.